scp/tests/test-structures/gen-operators/agent_gen_el.scs

```diff
@@ -0,0 +1,179 @@
+agent_gen_el
+<- abstract_sc_agent;
+=> nrel_primary_initiation_condition:
+    (sc_event_after_generate_outgoing_arc => action_initiated);
+=> nrel_sc_agent_action_class:
+    action_gen_el;
+=> nrel_initiation_condition_and_result:
+    (..agent_gen_el_initiation_condition
+        => ..agent_gen_el_result_condition);
+<= nrel_sc_agent_key_sc_elements:
+{
+    action_initiated;
+    action_gen_el
+};
+=> nrel_inclusion:
+    agent_gen_el_implementation
+    (*
+        <- platform_independent_abstract_sc_agent;;
+        <= nrel_sc_agent_program:
+        {
+            agent_gen_el_program
+        };;
+        -> agent_gen_el_active (* <- active_sc_agent;; *);;
+    *);;
+
+..agent_gen_el_initiation_condition
+= [*
+    action_gen_el _-> .._action;;
+    action_initiated _-> .._action;;
+    .._action _-> rrel_1:: .._set;;
+    concept_set _-> .._set;;
+*];;
+
+..agent_gen_el_result_condition
+= [*
+    .._set _-> .._generated_element;;
+*];;
+
+action_gen_el
+    <- sc_node_class;
+    <= nrel_inclusion: information_action;;
+
+agent_gen_el_program
+<- agent_scp_program;
+<- scp_program;
+-> rrel_key_sc_element: .._process;;
+
+agent_gen_el_program = [*
+@input_arc = (.._subscription_element _-> .._action);;
+.._process
+<-_ scp_process;
+
+_-> rrel_1:: rrel_in:: _event;
+_-> rrel_2:: rrel_in:: @input_arc;
+
+<=_ nrel_decomposition_of_action:: _...
+(*
+    _-> rrel_1:: .._operator1
+    (*
+        <-_ searchElStr3;;
+
+        _-> rrel_1:: rrel_assign:: rrel_scp_var:: .._subscription_element;;
+        _-> rrel_2:: rrel_fixed:: rrel_scp_const:: @input_arc;;
```

Comment:
```suggestion
.._process
<-_ scp_process;

_-> rrel_1:: rrel_in:: _event;
_-> rrel_2:: rrel_in:: _input_arc;

<=_ nrel_decomposition_of_action:: _...
(*
    _-> rrel_1:: .._operator1
    (*
        <-_ searchElStr3;;

        _-> rrel_1:: rrel_assign:: rrel_scp_var:: .._subscription_element;;
        _-> rrel_2:: rrel_fixed:: rrel_scp_const:: _input_arc;;
```

Reply:
why?

Reply:
no need to make @input_arc an arc, it can be _input_arc(node)